### PR TITLE
Feature/add random number methods

### PR DESCRIPTION
**Pull request by emekanwankwo**

### CODE REVIEW

File: app/routes/numberGeneration.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    // get numbers route
+    router.route('/getNumberIds')
+            .get(getPhoneNumberEntryIds);
+
+    // get numbers route
+    router.route('/getNumbers/:id')
+            .get(getPhoneNumbers);
+  };
+  
+  module.exports = numberGenerationRoute;
```
Comment (fortunee) [inline app/routes/numberGeneration.js]: You should add a line at the end of this file and any other file that doesn't have a line at the end.

File: public/pages/home-page/home-page.component.js [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+                return (order === 'desc') ? phoneNumber2 - phoneNumber1 : phoneNumber1 - phoneNumber2;
+            });
+        }
+
+        /**
+         * savePhoneNumbers
+         */
+        this.savePhoneNumbers = () => {
+            HomepageService.savePhoneNumbers(this.phoneNumbers, (response) => {
+              //
```
Comment (fortunee) [inline public/pages/home-page/home-page.component.js]: Is this suppose to be a comment? Get rid of it

**Review by fortunee — COMMENTED** [review]
Just a few comments. Everything else LGTM

**Comment by emekanwankwo** [comment]
Thanks @fortunee

**Pull request merged**